Modules/Filtering/ImageGrid/test/itkWarpVectorImageFilterTest.cxx

```diff
@@ -197,13 +197,14 @@ itkWarpVectorImageFilterTest(int, char *[])
   warper->SetOutputSpacing(array.GetDataPointer());
   ITK_TEST_SET_GET_VALUE(array, warper->GetOutputSpacing());
 
-  array.Fill(-10.0);
-  warper->SetOutputOrigin(array.GetDataPointer());
-  ITK_TEST_SET_GET_VALUE(array, warper->GetOutputOrigin());
-
-  array.Fill(0.0);
-  warper->SetOutputOrigin(array.GetDataPointer());
-  ITK_TEST_SET_GET_VALUE(array, warper->GetOutputOrigin());
```

Comment:
The take-home message seems to be: with C++20, ITK no longer supports a comparison like `array != point` (when `array` is an `itk::FixedArray`, and `point` is an `itk::Point`.)

It's OK to me, just compare a point to another point instead.

Reply:
HMMMM.  Perhaps we need to add this functionality for backward compatibility reasons?

I am overloaded with other obligations right now,  I'll leave this open for a bit to see if we are missing one of the following signatures in C++20:

```c++
bool itk::Point.opererator=(itk::FixedArray)
bool itk::FixkedArray.operator=(itk::Point)
```

Reply:
If you want to be able to compare an `itk::FixedArray` and an `itk::Point` by `itkFixedArray != itkPoint`, would you then also want to allow comparing an `itk::Vector` and an `itk::Point` that way?  Because both are publicly derived from `itk::FixedArray`. So you could then just use `operator!=(itk::FixedArray, itk::FixedArray)`

I feel that we lose type-safety, when allowing `itkVector == itkPoint` and `itkVector != itkPoint`.

Ideally, I think `itk::Point` should internally have a _private_ `itk::FixedArray`, instead of using public inheritance. Something to consider for ITK 6 or ITK 7  😺